src/main/resources/lib/menu.js

```diff
@@ -10,56 +10,56 @@ const globals = {
 
 /**
  * Returns the full breadcrumb menu path for the current content and site.
- * @param {Object} params - A JSON object containing the (optional) settings for the function.
+ * @param {Object} [params={}] - A JSON object containing the (optional) settings for the function.
  *   @param {Boolean} [params.linkActiveItem=false] - Wrap the active (current content) item with a link.
  *   @param {Boolean} [params.showHomepage=true] - Disable return of item for the site homepage.
  *   @param {String} [params.homepageTitle=null] - Customize (overwrite) the displayName of home/site link (if used). Common usage: "Home" or "Start".
  *   @param {String} [params.dividerHtml=null] - Any custom html you want appended to each item, except the last one. Common usage: '<span class="divider">/</span>'.
  *   @param {String} [params.urlType="Server"] - Control type of URL to be generated for menu items, default is 'server', only other option is 'absolute'.
- *   @param {String} [params.navigationAriaLabel=null] - The 'aria-label' attribute text on the '<nav>' element. This should be the name of the navigation, e.g "Breadcrumbs".
+ *   @param {String} [params.ariaLabel="breadcrumbs"] - The 'aria-label' attribute text on the '<nav>' element. This should be the name of the navigation, e.g "Breadcrumbs".
  * @returns {Object} - The set of breadcrumb menu items (as array) and needed settings.
  */
 exports.getBreadcrumbMenu = function (params = {}) {
-    let site = libs.portal.getSite();
-    let content = libs.portal.getContent() || site; // Fallback to site if there's no content (like in errorHandlers).
-
-    let breadcrumbItems = []; // Stores each menu item
-    let breadcrumbMenu = {}; // Stores the final JSON sent to Thymeleaf
-
     // Safely take care of all incoming settings and set defaults, for use in current scope only
     let settings = {
         linkActiveItem: params.linkActiveItem || false,
         showHomepage: params.showHomepage || true,
         homepageTitle: params.homepageTitle || null,
         dividerHtml: params.dividerHtml || null,
         urlType: params.urlType == "absolute" ? "absolute" : "server",
-        navigationAriaLabel: params.navigationAriaLabel || null,
+        ariaLabel: params.ariaLabel || "breadcrumbs",
     };
+    
+    const site = libs.portal.getSite();
+    const content = libs.portal.getContent() || site; // Fallback to site if there's no content (like in errorHandlers).
```

Comment:
A little sceptical to encouraging using this in errorhandlers. I think it's a good principle that in order to prevent loops of errorhandling when errorhandlers themselves fail, there should be as little logic as possible in errorhandlers that can concievably fail.

Reply:
Isent this exactly what we are doing by going back to site? 
How should we handle error handlers? 
Most people still want the 404 page to have a menu, but i suppose its possible to only return emtpy menu if it cant find the current content.

Reply:
It's still keeping (or recommending) some handling and functionality which might break. 

Keeping the careful principle would probably mean either having error-handler pages without menus, or hardcoding the menu to some extent - at least limiting the amount of processing as much as possible.